### PR TITLE
Add jade-data entry

### PR DESCRIPTION
**Pull request by rdebrand**
Preliminary jade-data record file.

### CODE REVIEW

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      "primary": "Dataset",
+      "secondary": [
+        "Collision"
+      ]
+    },
+    "usage": {
+      "description": "You can access these data through the EOS",
+      "links": [
+        {
+          "description": "Running CMS analysis code using Docker",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:72]: Note the "CMS" part which should not apply here.

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:8]: Shall we have an extra record with a list of JADE run numbers? If not, this link can be deleted.

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:4]: Please expand the text greatly. Since we shall have only one record for the JADE tape data, we need a very detailed description here.

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
+      "number_files": 2861,
+      "size": 2614994979
+    },
+    "doi": "10.7483/OPENDATA.9S5F.BY3B",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:37]: Please leave the DOI out for now. (This one would clash with an existing record.)  I'll reserve a new JADE-specific DOI when we shall be close to publishing.

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    "experiment": [
+      "JADE"
+    ],
+    "files": [
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:41]: I'll add the file information as we discussed.  (One file index for each tape.)

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+      }
+    ],
+    "keywords": [
+      "e+e- physics"
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p> During the experiment, the raw data was written to the tapes.</p>"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:56]: Any interesting links to point to for the methodology part? Perhaps a link to some JADE computing notes describing the process?

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "FIXME",
+    "title": "JADE raw data",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:60]: Please expand the title, it seems a bit too brief, especially considering that there will be only one record for all tapes. Something like "JADE detector raw data from electron-positron collision events at the PETRA accelerator".  (Quoting some text just for inspiration...)

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    "title_additional": "JADE raw data",
+    "type": {
+      "primary": "Dataset",
+      "secondary": [
+        "Collision"
+      ]
+    },
+    "usage": {
+      "description": "You can access these data through the EOS",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:69]: The usage part should be mostly how one can use the data, i.e. some VM or Docker images, links to analysis examples, etc. (The fact that the data can be accessed through EOS is not giving enough details for the visitors to this page; they will already see download links, but they would not know what to do with the files.)

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 7 above)
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/jade-data.json:2070]: This comment still holds, e.g. currently we have on the search pages:

jadetitle1

And on the detailed record page:

jadetitle2

It may not be necessary to use both `title` and `title_additional` fields. This was mostly useful for CMS which uses this field to store the exact dataset title in machine-readable format.  For the old JADE data, it would be enough to have only one `title` field, so that the title information in searches and on the detailed record page would be identical.

WDYT?

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 5 above)
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/jade-data.json:41]: Added.

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 8 above)
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: The usage guide now poitns to @andriish's personal GitHub repository. Shall we point to the JADE guide present on the portal instead? We have pages like https://opendata-qa.cern.ch/docs/jade-getting-started that should provide basic information how to get started with analysing JADE data.  (And these docs pages can then point to @andriish's repository!)

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 4 above)
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: I left the field empty.

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 7 above)
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: Sounds good, I moved the `title_additional ` to the main `title`.

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 8 above)
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: We now link the opendata site, but instead of `opendata-qa.cern.ch` we used `opendata.cern.ch`, is that fine?

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 9]
```diff
@@ -0,0 +1,2088 @@
...
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
```
[the remark this answers is not in the document]
Comment (andriish) [reply cernopendata/modules/fixtures/data/records/jade-data.json:33]: Hi @tiborsimko, the numbers came from a python script we are playing with. There was no 7M events in JADE :D.
I would suggest to have 0 events and then add the correct number in the next pull request. And take your numbers on size and space.

[the remark this answers is not in the document]
Comment (rdebrand) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: Set to 0 for now.

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 8 above)
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: This will work only when the data will be released in production. For now it's better to use relative links, e.g. "/docs/jade-getting-started", so that people could peruse things easily on QA. I'll change this during merge.

File: cernopendata/modules/fixtures/data/records/jade-data.json [hunk 10]
```diff
@@ -0,0 +1,2087 @@
...
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 0,
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/jade-data.json:33]: Why the change to the "0" as the number of events? The previous number 7017835 wasn't right? Will the number "0" be updated before the merge?

File: cernopendata/modules/fixtures/data/records/jade-data.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/jade-data.json]: As discussed IRL, I have also removed the three index files for the directories that have no data files: JAD, JAD016, JAD141.  Please have a look into those and we can eventually add them later.

```json
      {
        "checksum": "adler32:01d800c3",
        "size": 3,
        "type": "index.json",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD016_file_index.json"
      },
      {
        "checksum": "adler32:00000001",
        "size": 0,
        "type": "index.txt",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD016_file_index.txt"
      },
      {
        "checksum": "adler32:01d800c3",
        "size": 3,
        "type": "index.json",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD141_file_index.json"
      },
      {
        "checksum": "adler32:00000001",
        "size": 0,
        "type": "index.txt",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD141_file_index.txt"
      },
      {
        "checksum": "adler32:01d800c3",
        "size": 3,
        "type": "index.json",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD_file_index.json"
      },
      {
        "checksum": "adler32:00000001",
        "size": 0,
        "type": "index.txt",
        "uri": "root://eospublic.cern.ch//eos/opendata/jade/datasets/jadetapes/file-indexes/JAD_file_index.txt"
      }
```

**Review by tiborsimko — APPROVED** [review]
